timelock-impl/src/main/java/com/palantir/atlasdb/timelock/management/TimeLockManagementResource.java

```diff
@@ -101,6 +100,9 @@ public static TimeLockManagementService jersey(
 
     private static Set<PersistentNamespaceLoader> createNamespaceLoaders(
             PersistentNamespaceContext persistentNamespaceContext) {
+        if (persistentNamespaceContext.databasePersistence().isPresent()) {
+            return ImmutableSet.of(new DatabaseNamespaceLoader());
+        }
```

Comment:
Are we using the kvsConfig OR plan on using it? If so, It might be worth adding a comment otherwise, I am skeptical of passing around the config.